### DISCUSSION TITLE
Set that the avatar cannot be displayed normally

### DISCUSSION THREAD

**Discussion by zhimibuwudn** [misskey-dev/misskey#14488] [Q&A]
### 💡 Summary

Set that the avatar cannot be displayed normally, the minio object used is stored, the banner image is normal, and the avatar is abnormal. You can view the picture normally on the online disk.
`ERR  *	[server]	TimeoutError: Timeout awaiting 'request' for 60000ms
{
  code: 'BRIEF_REQUEST_INTERVAL',
  info: { remaining: 0, reset: 1725190083, resetMs: 1725190083498, total: 1 }
}
INFO *	[download]	Downloading https://****/misskey/files/4e17bab3-a165-42d9-9bf8-ee785d4d8805.jpg to /tmp/tmp-192-D1M2Dbb57xfR ...
INFO *	[download]	Downloading https://****/misskey/files/78981343-3661-47eb-94a8-19eee1bb2e5c.webp to /tmp/tmp-192-N2MjOUeeXfqz ...
INFO 1	[queue check-expired-mutings]	Checking expired mutings...
DONE 1	[queue check-expired-mutings]	All expired mutings checked.
INFO *	[download]	Downloading https://***/misskey/files/78981343-3661-47eb-94a8-19eee1bb2e5c.webp to /tmp/tmp-192-gzgnXR65nU75 ...
INFO *	[download]	Downloading https://***/misskey/files/4e17bab3-a165-42d9-9bf8-ee785d4d8805.jpg to /tmp/tmp-192-Gaw6KZPCWzor ...
ERR  *	[server]	TimeoutError: Timeout awaiting 'request' for 60000ms
ERR  *	[server]	TimeoutError: Timeout awaiting 'request' for 60000ms
ERR  *	[server]	TimeoutError: Timeout awaiting 'request' for 60000ms
ERR  *	[server]	TimeoutError: Timeout awaiting 'request' for 60000ms
{
  code: 'BRIEF_REQUEST_INTERVAL',
  info: { remaining: 0, reset: 1725190416, resetMs: 1725190416736, total: 1 }
}
INFO 1	[queue check-expired-mutings]	Checking expired mutings...
DONE 1	[queue check-expired-mutings]	All expired mutings checked.`

### 🥰 Expected Behavior

know

### 🤬 Actual Behavior

know

### 💻 Frontend Environment

```markdown
* Model and OS of the device(s):
* Browser:
* Server URL:
* Misskey:misskey/misskey:2024
```

### 🛰 Backend Environment (for server admin)

```markdown
* Installation Method or Hosting Service:
* Misskey:
* Node:
* PostgreSQL:
* Redis:
* OS and Architecture:
```

### Do you want to address this bug yourself?

- [ ] Yes, I will patch the bug myself and send a pull request

**Comment by zhimibuwudn** [comment]
![Snipaste_2024-09-01_18-29-51](https://github.com/user-attachments/assets/f718f846-b864-4f82-8358-cdb615c1c574)

**Comment by KisaragiEffective** [comment]
This needs more information.
* What is actual and exact version?
* Do you use some sort of fork or modify some code?
* Do you use container? If so, is the source https://hub.docker.com/r/misskey/misskey ?
* Do you use object storage? Are there any errors from the object storage provider?

**Reply by zhimibuwudn** [reply]
The official 2024.8.0 of Docker deployment
Use minio as the object storage. After uploading, the picture can be viewed normally, and it is normal to set the banner, but the avatar cannot be set.The official 2024.8.0 of Docker deployment
Use minio as the object storage. After uploading, the picture can be viewed normally, and it is normal to set the banner, but the avatar cannot be set.

**Comment by fruitriin** [comment]
may be cause by media proxy ( https://your.domain/proxy/avatar.webp?url=InFactYourImageUrl )
probably, it shows response error message or network errors on your console.

**Comment by zhimibuwudn** [comment]
It's normal to view and set banners on the network disk, but there is a problem with setting the avatar

**Comment by Pappmann** [comment]
I have exactly the same issue with misskey latest docker with podman.
